wled00/playlist.cpp

```diff
@@ -72,26 +126,30 @@ void loadPlaylist(JsonObject playlistObj) {
   currentPlaylist = 0; //TODO here we need the preset ID where the playlist is saved
 }
 
-void handlePlaylist()
-{
+
+void handlePlaylist() {
```

Comment:
i think it would be nice to stay with the prevalent code style of this repository. by that i mean leaving the curly braces where they've been. 
that also makes it easier to review this PR as there'd be less changes. :)

Reply:
Got that. But the code has all sorts of indentations and curly braces positions so I used the one that is more natural to me.